src/libnvme.map

```diff
@@ -175,6 +175,7 @@ LIBNVME_1_0 {
 		nvme_identify_secondary_ctrl_list;
 		nvme_identify_uuid;
 		nvme_init_copy_range;
+		nvme_init_copy_range_f1;
```

Comment:
This has to go a new section `LIBNVME_1_1`. I think we should also add this to the function documentation which which version a new function was introduced.

Reply:
Moved nvme_init_copy_range_f1 into the LIBNVME_1_1 section.  I will update the function documentation in a separate commit.